tests/components/emulated_hue/test_upnp.py

```diff
@@ -51,12 +52,14 @@ def test_description_xml(self):
         """Test the description."""
         result = requests.get(BRIDGE_URL_BASE.format("/description.xml"), timeout=5)
 
-        assert result.status_code == 200
+        assert result.status_code == HTTP_OK
         assert "text/xml" in result.headers["content-type"]
 
         # Make sure the XML is parsable
         try:
-            ET.fromstring(result.text)
+            root = ET.fromstring(result.text)
+            ns = {"s": "urn:schemas-upnp-org:device-1-0"}
+            assert root.find("./s:device/s:serialNumber", ns).text == "001788FFFE23BFC2"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)